### ISSUE TITLE
Can you add a  example -- operation  mutation

### ISSUE THREAD

**Issue by liaofangcai** [graphql-java/graphql-java#232]
how to use the method: new GraphQL(schema).execute("mutation sentence"). 
How to define the "mutation sentence".

**Comment by dminkovsky** [comment]
> How to define the "mutation sentence".

Do you mean: how do you construct mutation queries? If so, please see http://graphql.org/learn/queries/#mutations. I'm sorry that it's in English, but I am unaware of any non-English resources. 

If that's not what you mean, can you rephrase your question?

**Comment by liaofangcai** [comment]
Has been resolved, thank you very much！！！

**Comment by dminkovsky** [comment]
Great to hear!
